include/Page/Base.php

```diff
@@ -258,7 +259,8 @@ public function enqueue_script() {
 	 * Enqueue styles.
 	 */
 	public function enqueue_styles() {
-		wp_enqueue_style( 'bulk-move', $this->get_plugin_dir_url() . 'assets/css/bulk-move.css', array(), BM_BulkMove::VERSION );
+		wp_register_style( 'select2', $this->get_plugin_dir_url() . 'assets/css/select2.min.css' );
```

Comment:
Add jquery as dependency to `select2` library and also include its version. You can find the version that we are using from the `packages.json` file.